tests/testthat/test-package-installation-github.R

```diff
@@ -0,0 +1,38 @@
+# Run this test for users to make sure the github package
+# install feature of doAzureParallel are still working
+context("github package install scenario test")
+test_that("github package install Test", {
+  testthat::skip_on_travis()
+  credentialsFileName <- "credentials.json"
+  clusterFileName <- "cluster.json"
+
+  doAzureParallel::generateCredentialsConfig(credentialsFileName)
+  doAzureParallel::generateClusterConfig(clusterFileName)
+
+  # set your credentials
+  doAzureParallel::setCredentials(credentialsFileName)
+  cluster <- doAzureParallel::makeCluster(clusterFileName)
+  doAzureParallel::registerDoAzureParallel(cluster)
+
+  opt <- list(wait = TRUE)
+  '%dopar%' <- foreach::'%dopar%'
+  github <- c('azure/rAzureBatch', 'azure/doAzureParallel')
+  res <-
+    foreach::foreach(
+      i = 1:4,
+      .packages = c('httr'),
+      github = github,
+      .options.azure = opt
+    ) %dopar% {
+      "doAzureParallel" %in% rownames(installed.packages())
```

Comment:
This is actually an interesting test since rAzureBatch is installed as a dependency of doAzureParallel. Can you remove rAzureBatch from the list of github references, but still check that it is installed? May be worth doing that in a separate test though.

Reply:
Removed rAzureBatch but still check if it's installed in the test.